tenseal/__init__.py

```diff
@@ -76,6 +76,11 @@ def plain_tensor(*args, **kwargs) -> PlainTensor:
     return PlainTensor(*args, **kwargs)
 
 
+def plain_tensor_from(data: bytes) -> PlainTensor:
+    """Load a PlainTensor from a buffer."""
+    return PlainTensor.load_double(data)
```

Comment:
We can also have a load_int for integer tensor, rights? Should this be parameterize? Why not using protobuf so that we can also send this information (int/double) through?